src/schemas/tdc_time/TdcTime.h

```diff
@@ -0,0 +1,29 @@
+
+#pragma once
+
+#include "../../FlatBufferCreator.h"
+#include "../../SchemaRegistry.h"
+#include "../../helper.h"
+#include <map>
+#include <string>
+
+namespace TdcTime {
+class Converter : public FlatBufs::FlatBufferCreator {
```

Comment:
I think it would be useful to add your description from the PR description as a doxygen comment here:

> TDC forwarding module that ingests time as 2 x int32 and converts it into uint64 before putting it into a flat buffer of type tdct